### PR TITLE
improve article related list performance

### PR DESCRIPTION
**Pull request by chiangkeith**

### CODE REVIEW

File: src/components/article/LatestList.vue [hunk 1]
```diff
@@ -2,23 +2,27 @@
...
     <div class="title"><h4>最新文章</h4></div>
     <div class="item" v-for="(o, i) in latest" v-if="i < 6">
-      <div class="thumbnail" :style="{ backgroundImage: 'url(' + getValue(o, [ 'heroImage', 'image', 'resizedTargets', 'tiny', 'url'], '/asset/logo.png') + ')' }"></div>
+      <div class="thumbnail" :style="{ backgroundImage: 'url(' + getValue(o, [ 'heroImage', 'image', 'resizedTargets', 'tiny', 'url' ], '/asset/logo.png') + ')' }">
+        <a :href="`/post/${o.slug}`" target="_blank" :style="{ width: '100%', height: '100%', display: 'block' }"></a>
+      </div>
       <div class="content">
         <div class="content_category">
-          {{ getValue(o, [ 'categories', 0, 'title'], '國際') }}
+          <a :href="`/post/${o.slug}`" target="_blank">{{ getValue(o, [ 'categories', 0, 'title' ], '新聞') }}</a>
```
Comment (hcchien) [inline src/components/article/LatestList.vue]: The prefix might be 'post' or 'projects'.

File: src/components/article/LatestList.vue [hunk 2]
```diff
@@ -2,23 +2,27 @@
...
         </div>
         <div class="content_title">
-          <a href="">{{ o.title }}</a>
+          <a :href="`/post/${o.slug}`" target="_blank">{{ getTruncatedVal(o.title, 20) }}</a>
```
Comment (hcchien) [inline src/components/article/LatestList.vue]: same here.

File: src/components/article/RelatedList.vue [hunk 3]
```diff
@@ -1,9 +1,8 @@
 <template>
-  <div class="related-list-container">
+  <div class="related-list-container" v-if="ifshow">
     <div class="title"><h4>相關文章</h4></div>
     <div class="item" v-for="(o, i) in relateds">
-      <div class="title"><a :href="`/post/${getValue(o, [ 'name'], '')}`">{{ o.title }}</a></div>
-      <div class="brief"><a :href="`/post/${getValue(o, [ 'name'], '')}`">{{ getBrief(getValue(o, [ 'brief', 'apiData', 0, 'content', 0], '')) }}</a></div>
+      <div class="title"><a :href="`/post/${getValue(o, [ 'slug'], '')}`" target="_blank">{{ o.title }}</a></div>
```
Comment (hcchien) [inline src/components/article/RelatedList.vue]: same here.

File: src/components/article/LatestList.vue (hunk 1 above)
Comment (chiangkeith) [inline src/components/article/LatestList.vue]: revised, thanks

**Pull request merged**